### DISCUSSION TITLE
Keyboard shortcuts in the Ente Auth Windows application

### DISCUSSION THREAD

**Discussion by tzinm** [ente-io/ente#3986] [Enhancements]
After migrating from Authy a few days ago, there are some useful shortcuts which might be useful.

**Example 1**;

If a search returns more than one result, you can move through the different results with the ⬆ and ⬇ keyboard keys. Currently, these arrow keys place the course at the **beginning** or at the **end** of the search box respectively.

Once placed on the OTP code you want to copy with the arrow keys, just press the space bar and the code will be copied to the clipboard.

**Example 2**;

In case there is only **one search result**, a keyboard shortcut could be used to copy the OTP code directly. At the moment, the only way to do this via the keyboard is to use the tab key to move to the code and then press the space bar.

Thank you!

**Comment by woltersdylan** [comment]
I have Ente set to launch with $Mod + E and enabled focus search on launch. This is great for quickly jumping in and searching for the code i need. When I have one search result, pressing enter feels the most intuitive for copying the result, but also i tried CTRL+C at times. 

Spacebar feels non intuitive for copying, but I am guessing that is a side effect of the application framework being used.
